DEPLOYMENT_TYPES.md

```diff
@@ -15,9 +16,12 @@ The table below outlines the defaults per `deployment_type`.
 | **Image Streams**                                               | centos                                   | rhel + xpaas                           | N/A                              | rhel                             |
 
 
-**NOTE** `enterprise` deployment type is used for OpenShift Enterprise version
-3.0.x OpenShift Enterprise deployments utilizing version 3.1 and later will
-make use of the new `openshift-enterprise` deployment type.  Additional work to
-migrate between the two will be forthcoming.
+**NOTES**:
+
+- `enterprise` deployment type is used for OpenShift Enterprise version
+3.0.x.
+- OpenShift Enterprise deployments from version 3.1 to 3.?? use the `openshift-enterprise` deployment type.
 
 
+# REVIEW: notes, what versions use what
+# REVIEW: table, what is the registry_url for openshift-enterprise -- isn't it openshift3/ose-...?
```

Comment:
Yes it is `openshift3/ose-${component}:${version}`.

Although the AEP type would presumably still work with old versions, I think we can safely drop it from this table as no one will be consulting it for new installs. It would make this a lot easier to read.